### PR TITLE
Update to latest version of document. Use relative path and OntologyTerm object for ontology fields.

### PR DESCRIPTION
**Pull request by sdelatorrep**
- Update to the latest version of the [document](https://docs.google.com/document/d/171u8B-DXX58MgkEIRtoiV9lX6kmezVTW/edit#). 
- Use relative path (instead of absolute) for references in `$ref`
- All ontology fields are now an object compounded of 2 fields, `id` and `label`.

### CODE REVIEW

File: default_variant_in_sample_schema.yaml [hunk 1]
```diff
@@ -19,68 +19,90 @@ components:
...
+            - $ref: 'common.yaml#/components/schemas/OntologyTerm'
+            - example: 
+                id: 'ECO:0000006'
+                label: 'experimental evidence'
     VariantInSample:
       description: |
-        Todo
+        TBD
       type: object
+      required:
```
Comment (mbaudis) [inline default_variant_in_sample_schema.yaml:95]: Variant instances do not necessarily have a variant id. This has been discussed in another thread; so agreeing to this PR under the understanding that the later PR by @redmitry https://github.com/ga4gh-beacon/specification-v2-default-schemas/pull/27#issuecomment-824874275 will fix this.

File: default_variant_interpretation_schema.yaml [hunk 2]
```diff
@@ -19,118 +19,121 @@ components:
...
-          enum:
-            - benign
-            - likely benign
-            - pathogenic
-            - likely pathogenic
-            - unknown
-          example: pathogenic
+          allOf:
+            - description: |
+                Value from ICD10 disease codes or ontology terms from disease
```
Comment (mbaudis) [inline default_variant_interpretation_schema.yaml:30]: This specifies ICD10 as kind of special - I would reformulate this as one of the examples.

Comment (mbaudis) [reply default_variant_interpretation_schema.yaml:30]: ... this is *not* a blocking comment.

**Review by mbaudis — APPROVED** [review]
Approving this in general as improvement, with later fixes pending/to be developed.

**Review by jrambla — APPROVED** [review]

**Pull request merged**